src/plugins/share/public/components/context/index.tsx

```diff
@@ -35,6 +35,23 @@ export interface IShareContext extends ShareContext {
   anchorElement?: HTMLElement;
 }
 
-export const ShareTabsContext = createContext<IShareContext | null>(null);
+const ShareTabsContext = createContext<IShareContext | null>(null);
```

Comment:
This is a nit of existing code, but are we sure that `i18n` and `theme` are needed in the IShareContext interface? The ShareMenuProvider passes those services through, but those services should probably be acquired through core directly. Something for a future PR?